ex1.py

```diff
@@ -1,9 +1,9 @@
 #area of a circle
 import math
-def Area(r):
-    return((math.pi*r**2))
```

Comment:
why did you delete the function? i like this way of working. 
i know it is a small calculation. but i want you to get used to creating functions